Assets/Scripts/Commands/SnipStrokeCommand.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright 2020 The Tilt Brush Authors
```

Comment:
2022, Open Brush

Reply:
We so need to automate this!

Reply:
I was looking through the codebase and I think we missed a few elsewhere too 😅

Reply:
I thought we were only going to do new code. I don't mind doing a global search and replace if desired

Reply:
Ah, I mean on new file commits from the Open Brush authors. Gotta leave the credit to TB team!